pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -115,6 +115,28 @@ func skipMixerHTTPFilter(dir direction, mesh *meshconfig.MeshConfig, node *model
 	return disablePolicyChecks(dir, mesh, node) && mesh.GetDisableMixerHttpReports()
 }
 
+func (mixerplugin) OnOutboundPassthroughFilterChain(in *plugin.InputParams, mutable *networking.MutableObjects) error {
```

Comment:
I am seeing this add a  new method for the pass through filterchain only which was handles in `OnOutboundListener`.

Reply:
Great catch! With this PR the passthrough filter chain handled by `OnOutboundListener` is actually disabled. Let me update the PR to reflect it.

If you looked at how two listeners are merged before this PR, you probably have the same feeling with me:
1. The passthrough filter chains was not built correctly. It's good enough for per svc listener but doesn't work in single outbound listener.
2. The passthrough filter chain implementation was spread in `mergeTCPFilterChains`, `OnOutboundListener` and `appendListenerFallthroughRoute`.